pkg/user/handler.go

```diff
@@ -242,3 +243,29 @@ func (h Handler) FindById(c *gin.Context) {
 
 	c.JSON(http.StatusOK, userWithGroups)
 }
+
+// List user
```

Comment:
Now that you chose `GetAll` we should replace `List` with it everywhere 😄

Reply:
Please note [this](https://github.com/dhis2-sre/im-manager/pull/253#discussion_r1213896666) comment.

Reply:
I finally went with `FindAll` 😄